CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## v0.9.0
+**This version has breaking changes**
+* Eagerly load the `discovery` method so that multi-threaded applications do not face a race-condition when populating experiments.
```

Comment:
what do you think of?
```suggestion
* Eagerly load experiment definitions when booting rails, so that multi-threaded applications do not face a race-condition when populating experiments.
```